### DISCUSSION TITLE
Star Exchange

### DISCUSSION THREAD

**Discussion by sakshgupta** [Schweinepriester/github-profile-achievements#724] [Q&A]
[https://github.com/sakshgupta/Local_Seva](https://github.com/sakshgupta/Local_Seva)
Star my repo and I will do yours 🫂

**Comment by Schweinepriester** [comment]
See https://github.com/Schweinepriester/github-profile-achievements/discussions/722

**Comment by vinograd0v** [comment]


**Reply by sakshgupta** [reply]
> done https://github.com/mmanosalva/SublimeTeX

sry but i didn't see your star on my repo https://github.com/sakshgupta/Local_Seva
